packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar-menu-button/gux-table-toolbar-menu-button.scss

```diff
@@ -11,16 +11,15 @@
 }
 
 .gux-menu-button {
-  button {
-    color: var(--gse-ui-button-secondary-default-foregroundColor);
-    cursor: grab;
-    background-color: var(--gse-ui-button-secondary-default-backgroundColor);
-    border: none;
+  display: flex;
+  color: var(--gse-ui-button-secondary-default-foregroundColor);
+  cursor: pointer;
+  border: none;
 
-    gux-icon {
-      width: var(--gse-ui-icon-small-size);
-      height: var(--gse-ui-icon-small-size);
-    }
+  gux-icon {
+    align-self: center;
+    width: var(--gse-ui-icon-small-size);
+    height: var(--gse-ui-icon-small-size);
```

Comment:
```suggestion
    block-size: var(--gse-ui-icon-small-size);
    inline-size: var(--gse-ui-icon-small-size);
```

Reply:
linter should take care of this if you rebase on main